CoreScriptsRoot/Modules/Server/ClientChat/MessageLabelCreator.lua

```diff
@@ -67,6 +78,11 @@ local function WrapIntoMessageObject(id, BaseFrame, BaseMessage, Tweener, Strong
 	obj.Tweener = Tweener
 	obj.StrongReferences = StrongReferences
 	obj.UpdateTextFunction = UpdateTextFunction or function() warn("NO MESSAGE RESIZE FUNCTION") end
+	obj.ObjectPool = ObjectPool
+
+	if ObjectPool == nil then
+		print("Blah")
```

Comment:
I meant to remove this after debugging. Now there is no situation where this could happen.